application/src/main/java/org/opentripplanner/standalone/config/DebugUiConfig.java

```diff
@@ -0,0 +1,107 @@
+package org.opentripplanner.standalone.config;
+
+import static org.opentripplanner.standalone.config.framework.json.OtpVersion.V2_7;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.MissingNode;
+import java.io.Serializable;
+import java.util.List;
+import org.opentripplanner.standalone.config.debuguiconfig.BackgroundTileLayer;
+import org.opentripplanner.standalone.config.framework.json.NodeAdapter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class is an object representation of the 'debug-ui-config.json'.
+ */
+public class DebugUiConfig implements Serializable {
```

Comment:
Is this something we want to serialize? Isn't it enough to read this in at run time? Also, should we have some common interface/class for configs?

Reply:
I removed Serializable: 8395345b2483909862baf459c5fce0ae934f8bc8

I would love to refactor this class, but I request to do it in a separate PR.